store/storetest/post_store.go

```diff
@@ -699,9 +699,17 @@ func testPostStoreGetPostsWithDetails(t *testing.T, ss store.Store) {
 	_, err = ss.Post().Save(o6)
 	require.Nil(t, err)
 
-	r3, err := ss.Post().GetPosts(model.GetPostsOptions{ChannelId: o1.ChannelId, Page: 0, PerPage: 30}, false)
+	// Should only be 6 since we hit the cache
```

Comment:
Here you are re-introducing the cache tests that now makes no sense, because the cache is handled outside the sql store, we only need to undo your change manually. I'll push a commit with that.

Reply:
Done